src/locales/fr.json

```
@@ -422,13 +422,13 @@
     "logs_more": "Afficher plus de lignes",
     "search_for_apps": "Recherche des applications…",
     "unmaintained": "Non maintenue",
-    "purge_user_data_checkbox": "Purger les données de l'utilisateur %s ? (Cela supprimera le contenu de ses répertoires home et mail.)",
+    "purge_user_data_checkbox": "Purger les données de l'utilisateur %s ? (Cela supprimera le contenu de ses dossiers et répertoires dans home et mail.)",
     "purge_user_data_warning": "La purge des données de l’utilisateur n’est pas réversible. Assurez-vous de savoir ce que vous faites !",
     "version": "Version",
     "confirm_update_system": "Voulez-vous vraiment mettre à jour tous les paquets système ?",
     "app_state_inprogress_explanation": "Le responsable de cette application a indiqué qu'elle n'est pas encore prête pour une utilisation en production. SOYEZ VIGILANT(E) !",
     "app_state_notworking_explanation": "Le responsable de cette application a déclaré celle-ci comme \"non fonctionnelle\". SON INSTALLATION POURRAIT CASSER VOTRE SYSTÈME !",
-    "app_state_high-quality": "Haute qualité",
+    "app_state_high-quality": "niveau de qualité d'intégration de l'application : haute",
```

```suggestion
    "app_state_high-quality": "haute qualité",
```
 
This is supposed to be a label displayed in the catalog app cards so it can't be that long ...